src/main/target/common_pre.h

```diff
@@ -157,6 +157,13 @@
 #define DEFAULT_CPU_OVERCLOCK 0
 #endif
 
+#if defined(STM32H7)
+// Move ISRs to fast ram to avoid flash latency.
+#define HOT_IRQ_HANDLER FAST_CODE
```

Comment:
'HOT' and 'COLD' are terms used in optimization techniques are are nothing to do with the type of RAM the code lives in.

For examples, see the GCC documentation of `__attribute__ ((hot))` which does a different type of compile-time optimization for similar reasons.

https://gcc.gnu.org/onlinedocs/gcc-4.7.2/gcc/Function-Attributes.html

The terminology comes from the heat-map graphs you can create if you looked at every instruction, it's latency impact, and assigned a value based on the amount of times it was used and it's impact.  These IRQs are HOT, because calling them has a big impact and thus they should be optimized.  In this case the tecnique applied is moving them to, hopefully, faster ram, via the FAST_CODE macro and eventually selected linker sections and type of memory assigned to the linker section.

Reply:
OK, so HOT reflects a characteristic of the ISR, and FAST is our solution to addressing that. Our other use of FAST_CODE, on the filters for example, doesn't take this approach, declaring them to be HOT. Anywhere we see FAST currently we automatically understand that this code needs to run fast, most likely because it's time critical and called frequently. I still think that FAST is more consistent with current usage, and therefore clearer.

Reply:
Yeah, I'm not convinced that the use of `FAST` in the current codebase is a great name, I'll change it for you.

Reply:
btw, this is a PITA to change, as I have bunch of branches that add more HOT IRQs... 😢